docker/builder.dockerfile

```diff
@@ -39,11 +39,11 @@ RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
   && yarn install --frozen-lockfile --production --quiet \
   && rm -r "$YARN_CACHE_FOLDER"
 
-WORKDIR /workspace
```

Comment:
In the line above the yarn cache is deleted but later inside of `builder.sh` it seems that `python setup.py bdist_wheel --build-number 0` would benefit from the cache being around; no? Or it doesn't matter because we have`--modules-folder=/js/node_modules`?

Reply:
It doesn't matter because we already have the `node_modules` folder inside the image. This is tho save space on the built image.